### DISCUSSION TITLE
GitHub Copilot subscription issues

### DISCUSSION THREAD

**Discussion by leo-web3** [community/community#73497] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I paid for a one-year subscription to GitHub Copilot in September 2023, but suddenly I'm being told it has ended? I paid for a one-year plan, right? Why would it be discontinued after just two months? What is the problem?

**Comment by b8zeek** [comment, marked as answer]
You should definitely reach out to the GitHub support! This is not a case for the community to help.

Good luck! 🍺 

🍀

**Comment by davecheney** [comment]
@leo-web3 your subscription was cancelled because there was a problem processing your payment. We cannot assist you in this forum. Please contact support, https://support.github.com/contact.
